.gitignore

```diff
@@ -62,3 +62,6 @@ jade-dev-account.pem
 
 # Ignore generated credentials from google-github-actions/auth
 gha-creds-*.json
+
+# Ignore copied tanagra credentials
+rendered
```

Comment:
Is this required? I was hoping that you could grant access to the jade accounts so you could use the rendered json that TDR already requires (`/tmp/jade-dev-account.json`) instead of also relying on the tanagra SA.